PlayCover/en.lproj/Localizable.strings

```diff
@@ -86,6 +86,10 @@
 "settings.tab.km" = "Keymapping";
 "settings.toggle.km" = "Keymapping";
 "settings.toggle.km.help" = "Add keyboard support to touch-only games. Use Command + K to open editor menu.";
+"settings.button.km.download" = "Download keymapping";
+"settings.button.km.unavailable" = "No keymapping";
+"settings.button.km.loading" = "Loading...";
+"settings.button.km.forGame" = "Keymapping for";
```

Comment:
I think "Keymap" is more accurate.
And the Unavailable string better to be: "Not yet provided"
for Loading: "Searching..."

Reply:
I feel like "Keymapping" is fine as we were using it before.